src/memory_tracker-inl.h

```diff
@@ -81,6 +81,10 @@ void MemoryTracker::TrackFieldWithSize(const char* edge_name,
   if (size > 0) AddNode(GetNodeName(node_name, edge_name), size, edge_name);
 }
 
+void MemoryTracker::TrackInlineField(const MemoryRetainer& field) {
+  field.MemoryInfo(this);
```

Comment:
I feel like if we want to provide this as an utility, it might make sense to perform something similar to `Track()` and/or re-use it? That would be nice if we do want to see these fields as full “objects” on their own, and want to show them that way in heap dumps.

Then again, I think you’re making a good point about this approach being a good way to address the problem itself; I just didn’t think of a direct approach like this. :smile: Adding 2 calls à la `thread_stopper_.MemoryInfo(tracker);` in https://github.com/nodejs/node/pull/26099 would also be a good way to address this, as far as that PR is concerned in particular?